commercetools-internal-docs/src/main/java/io/sphere/sdk/meta/ReleaseNotes.java

```diff
@@ -144,6 +144,7 @@
  -->
  <h3 class=released-version id="v1_35_0">1.35.0 </h3>
  <ul>
+    <li class=fixed-in-release>Fixed field name in cart update action {@link io.sphere.sdk.carts.commands.updateactions.SetShippingRateInput}`, which was not properly working before</li>
```

Comment:
`fixed-in-release` is a very good classification: This is really a bug fix, because the update action never worked.